docs/server/source/appendices/all-in-one-bigchaindb.md

```diff
@@ -0,0 +1,84 @@
+# Run BigchainDB with all-in-one Docker
+
+**NOT for Production Use**
+
+For those who like using Docker and wish to experiment with BigchainDB in
+non-production environments, we currently maintain a BigchainDB all-in-one 
+Docker image and a
+`Dockerfile-all-in-one` that can be used to build an image for `bigchaindb`.
+
+This image contains all the services required for a BigchainDB node i.e.
+
+- BigchainDB Server
+- MongoDB
+- Tendermint
+
+*We understand that it is an anti-pattern to run multiple services inside a docker.*
```

Comment:
It makes sense to recommend a multi-image alternative, add a link to its description, and briefly mention its benefits. `it is an anti-pattern` is not really an explanation so I would omit it.